### PR TITLE
REF travis for transifex and last updates

### PR DESCRIPTION
**Pull request by eLBati**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,35 +1,44 @@
...
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" INCLUDE="sale_sourced_by_line_sale_transport_multi_address" LINT_CHECK="0"
+  global:
+  - VERSION="8.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  - TRANSIFEX_USER='transbot@odoo-community.org'
+  - secure: mTXYvzW7/+xP465dNW0p9H+Q8jVgQXmqm8zo+ntZC6ESkTudj6xUkrjJcfzp9LGomxoG6N+6TezjFuhfGKCAc4S2manxj9YQoHvZMHJvt73VsMZU2QLfE+IMAdFS/AeBpYLNE8VvdKIGhiFwvHyRqaF0vAGwmcsOdS2NQb/hoiY=
+
+  matrix:
+  - LINT_CHECK="1"
+  - TRANSIFEX="1"
+  - ODOO_REPO="odoo/odoo" EXCLUDE="sale_quotation_sourcing,sale_sourced_by_line,sale_quotation_sourcing_stock_route_transit,sale_sourced_by_line_sale_transport_multi_address" LINT_CHECK="0"
```
Comment (pedrobaeza) [inline .travis.yml]: You have to include here `TESTS="1"`

File: README.md [hunk 2]
```diff
@@ -13,3 +13,6 @@ This project aim to deal with modules related to manage sale and their related w
  - Easy the cancellation of SO
  - ...
 
+# Translation state
```
Comment (pedrobaeza) [inline README.md]:
```
Translation status
------------------
```

**Comment by eLBati** [comment]
@pedrobaeza thanks, fixed

**Comment by pedrobaeza** [comment]
Use `-----` for transifex status title

**Comment by pedrobaeza** [comment]
:+1:

Nitpicking: you don't need LINT_CHECK="0" in tests lines, because you already set it as global.

**Comment by guewen** [comment]
:+1:

**Comment by tafaRU** [comment]
:+1:

**Comment by tafaRU** [comment]
Someone of the MQT maintainers could also please tick the project here: https://github.com/OCA/maintainer-quality-tools/issues/194#issuecomment-107912200 ?

**Comment by StefanRijnhart** [comment]
@tafaRU done.

**Comment by tafaRU** [comment]
@StefanRijnhart thanks, but I think before to tick it we have to wait same changes for branch 7.0, isn'it?

**Comment by StefanRijnhart** [comment]
@tafaRU oh, alright then.

**Pull request merged**